### PR TITLE
Add refund info into faqs (some overlap with ecom faqs)

### PR DESCRIPTION
**Pull request by rebekaburnett**

### CODE REVIEW

**Review by cloveras — APPROVED** [review]
Nice! The eCom FAQ can probably refer to this, so it's as short as possible.

File: faqs/refunds-faq.md [hunk 1]
```diff
@@ -16,16 +16,22 @@ END_METADATA -->
...
-Document version 0.1.0.
+Document version 0.1.1.
 
 <!-- START_TOC -->
 
 ## Table of contents
 
 * [Can I refund a payment?](#can-i-refund-a-payment)
+* [How can I refund only a part of a payment?](#how-can-i-refund-only-a-part-of-a-payment)
+* [How long does it take from a refund is made until the money is in the customer's account?]
```
Comment (cloveras) [inline faqs/refunds-faq.md:27]: Needs a link.

File: faqs/refunds-faq.md [hunk 2]
```diff
@@ -36,6 +42,56 @@ All integrations with the Vipps eCom API _must_  support refunds.
...
+See:
+[For how long is a payment reserved?](reserve-and-capture-faq.md#for-how-long-is-a-payment-reserved).
+
+* The long version: It _is_ possible to cancel the remaining reservation after a
+partial capture through Vipps: Send a
+[`PUT:/ecomm/v2/payments/{orderId}/cancel`](https://vippsas.github.io/vipps-developer-docs/api/ecom#tag/Vipps-eCom-API/operation/cancelPaymentRequestUsingPUT)
+request with `shouldReleaseRemainingFunds: true` in the body.
+The payment must be `RESERVED` for this to take effect.
+See:
+[Cancelling a partially captured order](https://vippsas.github.io/vipps-developer-docs/docs/APIs/ecom-api/vipps-ecom-api#cancelling-a-partially-captured-order).
```
Comment (cloveras) [inline faqs/refunds-faq.md:71]: I'm not sure about this, but should we indicate that the link is to the eCom API, and not general info for all APIs?

```suggestion
[Vipps eCom API: Cancelling a partially captured order](https://vippsas.github.io/vipps-developer-docs/docs/APIs/ecom-api/vipps-ecom-api#cancelling-a-partially-captured-order).
```

File: faqs/refunds-faq.md (hunk 1 above)
Comment (rebekaburnett) [reply faqs/refunds-faq.md]: Oops!!! fixed.

**Pull request merged**